ajax/libs/pipes-core/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "pipes-core",
+  "version": "0.8.4",
+  "description": "Basic utilities for web streams",
+  "filename": "dist/pipes.core.min.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/pipesjs/core.git"
+  },
+  "keywords": [
+    "streams",
+    "pipesjs",
+    "utils"
+  ],
+  "author": "Diwank Singh <diwank.singh@gmail.com>",
+  "license": "MIT",
+  "homepage": "https://pipes.js.org/core"
+}
```

Comment:
Let's add auto update fields to let cdnjs be able to catch the library's future versions.
They are as follows:
if you want to use npm as the source of the lib.
```
"npmName": "@pipes/core",
  "npmFileMap": [
    {
      "basePath": "dist",
      "files": [
        "pipes.core.!(es6)*"
      ]
    }
  ]
```

If you want to use GitHub as the source.
```
"autoupdate": {
    "source": "git",
    "target": "git://github.com/pipesjs/core.git",
    "fileMap": [
      {
        "basePath": "dist",
        "files": [
          "pipes.core.!(es6)*"
        ]
      }
    ]
  }
```

To know more about auto update, you can refer https://github.com/cdnjs/cdnjs/blob/master/documents/autoupdate.md.